cmd/exchange_template/exchange_template.go

```diff
@@ -108,8 +108,8 @@ func main() {
 	}
 
 	fmt.Println("GoCryptoTrader: Exchange templating tool service complete")
+	fmt.Println("Add appropriate exchange config settings, particularly enabled and available assets and pairs, to config_example.json for proper functionality (it will automatically be added to testdata/configtest.json)")
```

Comment:
Feels like this could be a good time to make this whole section a single multi-line block, no?
If so, that applies to L73-78 as well.

Reply:
Do you think it is necessary to put it into a single Println() file here?